theme_override/mkdocs.yml

```diff
@@ -0,0 +1,48 @@
+site_name: OpenShift as a Service
+docs_dir: content
+site_url: https://docs.stakater.com/oaas/
+repo_url: https://github.com/stakater/oaas-docs
+edit_uri: blob/main/content/
+use_directory_urls: false
+theme:
+  name: material
+  logo: assets/images/favicon.svg
+  favicon: assets/images/favicon.svg
+  custom_dir: dist/_theme
+  features:
+    - content.code.copy
+    - content.code.select
+
+markdown_extensions:
+  - admonition
+  - pymdownx.details
+  - pymdownx.superfences:
+      custom_fences:
+        - name: mermaid
+          class: mermaid
+          format: !!python/name:pymdownx.superfences.fence_code_format
+  - attr_list
+  - pymdownx.emoji:
+      emoji_index: !!python/name:material.extensions.emoji.twemoji
+      emoji_generator: !!python/name:material.extensions.emoji.to_svg
```

Comment:
some props look like already exist in common. double check.

Reply:
Same: The issue with the merge script still exists for list properties, there's another ticket open for this issue that was raised earlier. We need to look at that, for theme common in general.